### ISSUE TITLE
Extra spacing added to single top level unordered list

### ISSUE THREAD

**Issue by SteveALee** [prettier/prettier#6481]
I can't use the play ground as it does not support MD. At least I can find no way to sepcify the language.

In visuall studio code I am finding that prettier adds spaces aftet the `-` character starting a single top level list.
Even though PR #4893 indicates this should have been fixed. I coldn't easily see if that PR had been reelased bu dates would inidcate it should ahve.

```
# Top level

- aaa
- bbb
```

is converted to the following on save

```md
# Top level

-   aaa
-   bbb
```

which is horrible and prettier lint extension moans with it's default rules - which I'm happy with

.prettierrc.json
```
{
	"trailingComma": "all",
	"semi": false,
	"singleQuote": true,
	"useTabs": true,
	"tabWidth": 4,
	"endOfLine": "lf"
}
```

.editorcong (haveduplication as the Svelte extension for code ignores .editorconfig :(
```
# editorconfig.org
root = true

[*]
indent_style = tab
indent_size = 4
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false
```

- "prettier": "1.18.2",
- "prettier-plugin-svelte": "0.7.0",
- Prettier -extension for code esbenp.prettier-vscode 2.2.2

**Comment by SteveALee** [comment]
I guess the issue is I do not think the indent level should apply here. Indeed there is no indent separate from tab as far as I can see

**Comment by lydell** [comment]
Hi!

This is how Prettier formats your input code:

**Prettier 1.18.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiABAFQgB3QGzgDc58AdKCgWnQEN7r0AjFkAGhFxgEtoBnZKFoAnYRADuABREIBKWkQjcAJuxBNhtMAGs4MAMo4t3KAHNkMYQFc4HABYwAtvgDqd7vD5GwcfbI-cRB4AnsjgfAIcJnxwwjCSmqaOtMgAZrT4MRwAVnwAHgBCmjp6+rSOcAAyJnBpGVkguXn6JqaEAIpWEPB1mbYgRsIxwmHJwtrKElBqOMImMC4qMHbIABwADByzEDEumjhhs3DDJGoAjl3wCbhyILR8VFBwcMovasJwF9wfCbRJKUh0n0ODFHNwLNZ+nxWh1LrVAfV+jBaExFsplsgAEwcSy0bj4VoAYQgjmSYSg0FqHCsMUwKLkQJiAF8mUA)
```sh
--parser markdown
```

**Input:**
```markdown
# Top level

- aaa
- bbb
```

**Output:**
```markdown
# Top level

- aaa
- bbb

```

In other words – the output is identical to the input.

**Comment by SteveALee** [comment]
aha so that's how I select language! Obvious now

So try this [playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiABAFQgB3QGzgDc58AdKCgWnQEN7r0AjFkAGhFxgEtoBnZKFoAnYRADuABREIBKWkQjcAJuxBNhtMAGs4MAMo4t3KAHNkMYQFc4HABYwAtvgDqd7vD5GwcfbI-cRB4AnsjgfAIcJnxwwjCSmqaOtMgAZrT4MRwAVnwAHgBCmjp6+rSOcAAyJnBpGVkguXn6JqaEAIpWEPB1mbYgRsIxwmHJwtrKElBqOMImMC4qMHbIABwADByzEDEumjhhs3DDJGoAjl3wCbhyILR8VFBwcMovasJwF9wfCbRJKUh0n0ODFHNwLNZ+nxWh1LrVAfV+jBaExFsplsgACwcSy0bj4VoAYQgjmSYSg0FqHCsMUwKLklhsAF8mUA) which has the indent settings I am using

I guess it's a mute point if the items should follow tab stops for intital indent or not treat thuis as an indent so i'll close this issue..

My workaround was to use an override for md in my prettier config with tab spacing of 2. Hopefull I'll never actuall need tabs in the markdown or I'll be stuck.

Thanks
